torchvision/transforms/functional.py

```diff
@@ -392,7 +392,7 @@ def resize(
             :class:`torchvision.transforms.InterpolationMode`.
             Default is ``InterpolationMode.BILINEAR``. If input is Tensor, only ``InterpolationMode.NEAREST``,
             ``InterpolationMode.BILINEAR`` and ``InterpolationMode.BICUBIC`` are supported.
-            For backward compatibility integer values (e.g. ``PIL.Image.NEAREST``) are still acceptable.
+            For backward compatibility integer values (e.g. ``PIL.Image(.Resampling).NEAREST``) are still acceptable.
```

Comment:
I think we should deprecate this.

cc @NicolasHug @pmeier

Reply:
@vfdev-5

Reply:
I'm OK with deprecating this.

Reply:
@kylematoba Would you be up to send a follow-up PR that deprecates this?